aiohttp/client_exceptions.py

```diff
@@ -152,6 +152,27 @@ class ClientProxyConnectionError(ClientConnectorError):
     """
 
 
+class UnixClientConnectorError(ClientConnectorError):
+    """Unix connector error.
+
+    Raised in :class:`aiohttp.connector.UnixConnector` if
+        connection to unix socket can not be established.
+    """
+    def __init__(self, path: str, connection_key: ConnectionKey,
+                 os_error: OSError) -> None:
+        self._path = path
+        super().__init__(connection_key, os_error)
+
+    @property
+    def path(self) -> str:
+        return self._path
+
+    def __str__(self) -> str:
+        return ('Cannot connect to unix socket {0.path} ssl:{1} [{2}]'
```

Comment:
I have two comments here:
1) it's better to use named placeholders instead of positional ones
2) you could do `self.path` in args like with other vars, otherwise, it looks inconsistent